Library/Formula/git-stree.rb

```diff
@@ -0,0 +1,15 @@
+class GitStree < Formula
+  homepage "https://github.com/tdd/git-stree"
+  head "https://github.com/tdd/git-stree.git"
+  url "https://github.com/tdd/git-stree/archive/v0.4.0.tar.gz"
+  sha256 "c04d19c8eb1a566a8e7f373d447ab7e3480ee95f94802fa957afeb22c4bed0ef"
+
+  def install
+    bin.install("git-stree")
+    bash_completion.install("git-stree-completion.bash" => "git-stree")
+  end
+
+  test do
+    shell_output("git stree help 2>&1").include?("Usage:")
```

Comment:
No, I'd rather not have the test pull someone's git repo as a subtree just for a test.